deployment/modules/cloudflare/tiles/workers.tf

```diff
@@ -2,20 +2,25 @@ data "cloudflare_zone" "immich_cloud" {
   name = "immich.cloud"
 }
 
-resource "cloudflare_worker_script" "tiles" {
+resource "cloudflare_workers_script" "tiles" {
   account_id = var.cloudflare_account_id
   name       = "tiles"
-  content    = file("${path.module}/dist/tiles.js")
+  content    = file("${var.tiles_build_dir}/index.js")
   module     = true
 
   plain_text_binding {
-    name = "CACHE_CONTROL"
-    text = "public, max-age=2678400" // 31 days - cloudflare cdn maximum
```

Comment:
Do we no longer need the cache control?

Reply:
We no longer control it from a variable, it's just hardcoded